content/en/docs/tasks/job/parallel-processing-expansion.md

```diff
@@ -1,51 +1,66 @@
 ---
 title: Parallel Processing using Expansions
-content_template: templates/concept
+content_template: templates/task
 min-kubernetes-server-version: v1.8
 weight: 20
 ---
 
 {{% capture overview %}}
 
-In this example, we will run multiple Kubernetes Jobs created from
-a common template.  You may want to be familiar with the basic,
-non-parallel, use of [Jobs](/docs/concepts/workloads/controllers/jobs-run-to-completion/) first.
+This task demonstrates running multiple {{< glossary_tooltip text="Jobs" term_id="job" >}}
+based on a common template. You can use this approach to process batches of work in
+parallel.
 
+For this example there are only three items: _apple_, _banana_, and _cherry_.
+The sample Jobs process each item simply by printing a string then pausing.
+
+See [using Jobs in real workloads](#using-jobs-in-real-workloads) to learn about how
+this pattern fits more realistic use cases.
 {{% /capture %}}
 
+{{% capture prerequisites %}}
 
-{{% capture body %}}
+You should be familiar with the basic,
+non-parallel, use of [Job](/docs/concepts/jobs/run-to-completion-finite-workloads/).
 
-## Basic Template Expansion
+{{< include "task-tutorial-prereqs.md" >}}
 
-First, download the following template of a job to a file called `job-tmpl.yaml`
+For basic templating you need the command-line utility `sed`.
 
-{{< codenew file="application/job/job-tmpl.yaml" >}}
+To follow the advanced templating example, you need a working install of
```

Comment:
nit: ... you need a working installation of ... OR
... you first need to install ...